src/chains/celo/serializers.test.ts

```diff
@@ -475,6 +460,21 @@ describe('cip64', () => {
     expect(parseTransactionCelo(tx1)).toEqual(parseTransactionCelo(tx2))
     expect(parseTransactionCelo(tx1)).toEqual({ ...baseCip64, type: 'cip64' })
   })
+
+  test('CIP-42 transaction that has all CIP-64 fields and CIP-64 takes precedence', () => {
```

Comment:
so this is saying only only process as cip42 when gatewayFeeRecipient is present

otherwise never do 

correct?